src/app/pages/tabs-links-page/tabs-links-page.component.ts

```diff
@@ -7,8 +9,17 @@ import { BreadcrumbsComponent } from 'src/app/components/breadcrumbs/breadcrumbs
   styleUrls: ['./tabs-links-page.component.css'],
   standalone: true,
   schemas: [CUSTOM_ELEMENTS_SCHEMA],
-  imports: [BreadcrumbsComponent],
+  imports: [BreadcrumbsComponent, RouterLink, RouterOutlet, CommonModule],
 })
 export class TabsLinksPageComponent {
+  fullPath: string;
 
+  constructor(private router: Router, private location: Location) {
```

Comment:
It is usually good practice to shut it down the subscription. 

Even this component is destroyed (a.k.a detached from the DOM), subscription will stay active. 

there are different  ways to close a subscriptions. 

I am going to add re-usable method (in util file) to shut down a subscription when component is destroyed. You can check usage if you need it later :)